src/ffjord.jl

```diff
@@ -159,6 +159,7 @@ function jacobian_fn(f, x::AbstractMatrix)
     for i in 1:size(y, 1)
         Zygote.@ignore z[i, :] .+= one(eltype(x))
         vec[i, :, :] = back(z)[1]
+        Zygote.@ignore fill!(z, zero(eltype(x)))
```

Comment:
It probably doesn't matter much, but it's probably best to just zero the parts that changed, i.e. `Zygote.@ignore z[i, :] .= zero(eltype(x))`